lib/internal/webstreams/readablestream.js

```diff
@@ -681,6 +683,16 @@ class ReadableStreamBYOBRequest {
         'This BYOB request has been invalidated');
     }
 
+    const viewedBuffer = ArrayBufferViewGetBuffer(view);
+    const viewedBufferByteLength = ArrayBufferGetByteLength(viewedBuffer);
+    const isDetached = arrayBufferIsDetached(viewedBuffer);
```

Comment:
Does this have to be a call into C++ (which is generally slow)? I was wondering if we could try out the existing validations that are used elsewhere:
* https://github.com/nodejs/node/blob/7a9266222dd07f49d29f42a24b07238cbc4c9fc9/lib/internal/webstreams/readablestream.js#L893-L908
* https://github.com/nodejs/node/blob/7a9266222dd07f49d29f42a24b07238cbc4c9fc9/lib/internal/util/inspect.js#L1689-L1695

Reply:
Indeed. It looks better approach. Applied the suggestions, PTAL.